includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -118,6 +120,25 @@ public function admin_options( WC_Stripe_Payment_Gateway $gateway ) {
 		echo $account_data_exists ? '<div id="wc-stripe-account-settings-container"></div>' : '<div id="wc-stripe-new-account-container"></div>';
 	}
 
+	/**
+	 * Determines if the payments task is completed in WooCommerce onboarding flow.
+	 *
+	 * @return bool True if the payments task is completed, false otherwise.
+	 */
+	private function is_payments_onboarding_task_completed(): bool {
```

Comment:
Note: `TaskLists` have a data store action to retrieve tasks in the [frontend](https://github.com/woocommerce/woocommerce/blob/b45221c22987cae5b871139f58baff5fcf1b103b/plugins/woocommerce/client/admin/docs/features/onboarding-tasks.md#frontend), but I am always getting an empty list when using it. So I've resorted to checking it on the backend.